src/deconvolution/demuxlet/config.vsh.yaml

```diff
@@ -0,0 +1,56 @@
+functionality:
```

Comment:
Could you add freemuxlet and demuxlet to a separate PR? This way the commit history is a bit cleaner.